CMakeLists.txt

```diff
@@ -18,13 +18,13 @@ add_flag_source(src/shared/XLinkDevice.c "-Werror=switch-enum")
 
 if(NOT WIN32)
     target_include_directories(${TARGET_NAME}
-            PRIVATE
-            "${LIBUSB_INCLUDE_DIR}")
+        PRIVATE "${LIBUSB_INCLUDE_DIR}"
+    )
 
     target_link_libraries(${TARGET_NAME}
-            PUBLIC
-            Threads::Threads
-            ${LIBUSB_LIBRARY})
+        PRIVATE Threads::Threads
+        PUBLIC ${LIBUSB_LIBRARY}
+    )
```

Comment:
Intentional change?

Reply:
unintentioned, not commit this, maybe over the merge with develop branch.